deluge/ui/console/cmdline/commands/config.py

```diff
@@ -54,11 +54,11 @@ def atom(src, token):
     elif token[1].lower() == 'false':
         return False
     elif token[0] is tokenize.STRING or token[1] == '/':
-        return token[-1].decode('string-escape')
+        return token[-1].encode('utf-8').decode('unicode-escape')
```

Comment:
This replacement does not feel correct so I did some research, see https://stackoverflow.com/a/24519338/175584

My conclusion is that we remove the decode with string-escape as I can't see which escapes (newline, tab, etc.) would actually be needed for console commmands. Do you agree?

However if later on someone feels that escaping is useful then it can be revisited with concrete examples and unit tests using the regex function from that SO answer.

Reply:
I found the way the functionality a bit weird overall but wanted to stay true to the original design and just make it work in Python 3.

Yes, I agree that it can just be removed for now, not sure how widely used it is anyways and newlines and such in a config setting isn't common.